.github/workflows/unit-tests.yml

```diff
@@ -73,6 +73,14 @@ jobs:
             job_label_php: 'current'
             job_label_wordpress: 'latest'
 
+            # Temporary WordPress 6.7 testing (PMCP-7054)
+          - php: '8.2'
+            wordpress: '6.7'
```

Comment:
What's our timeline for the upgrade? 6.8 is out next month, should we focus on that instead?